Zotero/Scenes/Detail/DetailCoordinator.swift

```diff
@@ -471,7 +469,7 @@ extension DetailCoordinator: DetailItemsCoordinatorDelegate {
             showNote(library: viewModel.state.library, kind: .standaloneCreation(collection: viewModel.state.collection), saveCallback: nil)
         }))
 
-        if viewModel.state.library.metadataAndFilesEditable {
+        if viewModel.state.library.filesEditable {
```

Comment:
If metadata editing is disabled, you can't create an item for the new file attachment? So why go from `metadataAndFilesEditable` to just `filesEditable`?